src/lib/crypto/kimchi_bindings/stubs/kimchi_types.ml

```diff
@@ -16,15 +16,18 @@ type nonrec 'caml_f random_oracles =
   ; v_chal : 'caml_f scalar_challenge
   ; u_chal : 'caml_f scalar_challenge
   }
+[@@boxed]
```

Comment:
what's this for? i also get these when i build the repo

Reply:
I think this is caused by a change in the ocaml-gen version. Your cargo cache might be stale; as usual, I blame nix.